Builds/CMake/RippledSanity.cmake

```diff
@@ -72,10 +72,26 @@ if ("${CMAKE_CURRENT_SOURCE_DIR}" STREQUAL "${CMAKE_BINARY_DIR}")
     "directory from ${CMAKE_CURRENT_SOURCE_DIR} and try building in a separate directory.")
 endif ()
 
-if ("${CMAKE_GENERATOR}" MATCHES "Visual Studio" AND
+if ("${CMAKE_GENERATOR}" MATCHES "Visual Studio.*2017" AND
     NOT ("${CMAKE_GENERATOR}" MATCHES .*Win64.*))
   message (FATAL_ERROR
-    "Visual Studio 32-bit build is not supported. Use -G\"${CMAKE_GENERATOR} Win64\"")
+    "Visual Studio 32-bit build is not supported. Use -G\"${CMAKE_GENERATOR} Win64\". "
```

Comment:
I don't think this format is supported going forward. Because we require CMake 3.9+, we can comfortably ask users to pass `-G \"${CMAKE_GENERATOR}\" -A x64`.

Reply:
You're right about the old (VS 2017) format not being supported going forward, which is why I split them out the way I did. What I _didn't_ realize or check is that the old generator works with the `-A` option.

I don't know how if it'll cause any trouble for others, but I can update to remove this format and see what happens.

Reply:
Update: I left it backward compatible, but simplified to only show the message requesting the `-A` option.